cartridge-cli.lua

```diff
@@ -1076,8 +1084,8 @@ local function generate_version_file(source_dir, dest_dir, app_name, app_version
         version_file:write(string.format('TARANTOOL=%s\n', _TARANTOOL))
     end
 
-    local _, _, app_commit = detect_version(source_dir)
-    version_file:write(string.format("%s=%s-%s\n", app_name, table.concat(app_version, '.'), app_commit or ''))
+    local _, _, app_commit = detect_git_version(source_dir)
```

Comment:
I think it shouldn't be called here.

We already detected git version and saved that.

I suggest 

```lua
function generate_version_file(source_dir, dest_dir, app_name, app_version, app_release)
...
version_file:write(string.format("%s=%s-%s\n", app_name, app_version, app_release)
...
end
...

```

```

Reply:
I hope I nothing break

Fixed